WordPress/Sniff.php

```diff
@@ -42,37 +41,37 @@
 	 * @var array
 	 */
 	public static $escapingFunctions = array(
-		'absint' => true,
-		'esc_attr__' => true,
-		'esc_attr_e' => true,
-		'esc_attr_x' => true,
-		'esc_attr' => true,
-		'esc_html__' => true,
-		'esc_html_e' => true,
-		'esc_html_x' => true,
-		'esc_html' => true,
-		'esc_js' => true,
-		'esc_sql' => true,
-		'esc_textarea' => true,
-		'esc_url_raw' => true,
-		'esc_url' => true,
-		'filter_input' => true,
-		'filter_var' => true,
-		'intval' => true,
-		'json_encode' => true,
-		'like_escape' => true,
-		'number_format' => true,
-		'rawurlencode' => true,
-		'sanitize_html_class' => true,
-		'sanitize_user_field' => true,
-		'tag_escape' => true,
-		'urlencode_deep' => true,
-		'urlencode' => true,
-		'wp_json_encode' => true,
+		'absint'               => true,
+		'esc_attr__'           => true,
+		'esc_attr_e'           => true,
+		'esc_attr_x'           => true,
+		'esc_attr'             => true,
+		'esc_html__'           => true,
+		'esc_html_e'           => true,
+		'esc_html_x'           => true,
+		'esc_html'             => true,
+		'esc_js'               => true,
+		'esc_sql'              => true,
+		'esc_textarea'         => true,
+		'esc_url_raw'          => true,
+		'esc_url'              => true,
+		'filter_input'         => true,
+		'filter_var'           => true,
+		'intval'               => true,
+		'json_encode'          => true,
+		'like_escape'          => true,
+		'number_format'        => true,
+		'rawurlencode'         => true,
+		'sanitize_html_class'  => true,
+		'sanitize_user_field'  => true,
+		'tag_escape'           => true,
+		'urlencode_deep'       => true,
+		'urlencode'            => true,
+		'wp_json_encode'       => true,
 		'wp_kses_allowed_html' => true,
-		'wp_kses_data' => true,
-		'wp_kses_post' => true,
-		'wp_kses' => true,
+		'wp_kses_data'         => true,
+		'wp_kses_post'         => true,
+		'wp_kses'              => true,
```

Comment:
Honestly, I'm not big on these array alignment. It can make things look better, but IMO it isn't necessary when all of the values are currently the same, and it will inevitably get out of sync next time a longer function is added to the list. When that happens we're either going to realign the whole array, or we'll end up with a partly aligned array, which after a while will put us back in square one. :-) I'm not going to complain if others disagree, but I just thought I'd mention it in case anybody wants to give a second opinion.

For the record, the variable alignments look good, I'm just talking about these array alignments.

Or do we still sniff for array alignments? I don't think that we do. Or else maybe I've disabled that (but I don't think so)...